README.md

```diff
@@ -1,2 +1,46 @@
 # platform-starter
 Starter and basic configuration for platform frontend projects.
+
+### Requirements
+
+* Go installed
+* Node.js installed
+* NPM or yarn installed (yarn preferred)
```

Comment:
For example in my machine `yarn global add` does not work. Having the ability to install via npm is nice, I think.